core-scroll-header-panel.css

```diff
@@ -26,6 +26,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+/* github.com/Polymer/core-scroll-header-panel/pull/38*/
+#mainContainer {
+  transform: translateZ(0);
```

Comment:
This should also have a -webkit-transform version before it. Safari, Mobile Safari, and Android Browser still use the prefix: http://caniuse.com/#search=transform